boards/Makefile.common

```diff
@@ -164,3 +165,6 @@ clean::
 
 .PHONY: debug
 debug: target/$(TARGET)/debug/$(PLATFORM).elf
+
+.PHONY: release
+debug: target/$(TARGET)/release/$(PLATFORM).elf
```

Comment:
Presumably this should be `release:`

Also, why do we have these targets in two places (also line 108)? That seems odd.

Reply:
You're right. fixing.